src/app/components/InternalLinkCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  Bullseye, Button, Card, CardBody, CardFooter, CardHeader, CardTitle,
+  Flex, FlexItem, Title
+} from '@patternfly/react-core'
+
+
+import redhat_logo from '@app/bgimages/redhat_clear.png'
+
+
+const providerInfo = {
+  name: 'redhat',
+  title: 'Build your own RHEL cloud image',
+  description: "With Red Hat's image builder, you can create customizable and repeatable operating system images and server images",
+  text: 'Build an image ->',
+  url: 'https://console.redhat.com/insights/image-builder'
+}
+
+const InternalLinksCard: React.FunctionComponent = () => {
+  return (
+    <Flex direction={{default: 'column'}} flex={{default: 'flex_1'}}
+          justifyContent={{default: 'justifyContentSpaceBetween'}}
+          alignSelf={{default: "alignSelfStretch"}}
+          style={{border: '1px solid #D2D2D2', padding: '2%'}}>
```

Comment:
Here I think you ought to also use the Card component:
https://www.patternfly.org/v4/components/card